README.md

```diff
@@ -73,15 +73,37 @@ After the install process is over, you'll be able to run your application using
 $ grunt
 ```
 
-Your application should run on port 3000, so in your browser just go to [http://localhost:3000](http://localhost:3000)
+Your application should run on port 3000 with the *development* environment configuration, so in your browser just go to [http://localhost:3000](http://localhost:3000)
 
 That's it! Your application should be running. To proceed with your development, check the other sections in this documentation.
 If you encounter any problems, try the Troubleshooting section.
 
+* explore ```config/env/development.js``` for development environment configuration options
```

Comment:
We *have* to use 3 ticks for code blocks, but for inline strings 1 tick can be used.